helm/prometheus-rules/templates/alerting-rules/job.rules.yml

```diff
@@ -14,14 +14,27 @@ spec:
     - alert: ManagementClusterJobFailed
       annotations:
         description: '{{`Job {{ $labels.namespace }}/{{ $labels.job_name }} is failed.`}}'
-      expr: kube_job_failed{cluster_type="management_cluster", condition="true"} == 1
+      expr: kube_job_failed{cluster_type="management_cluster", condition="true", job_name!~"silence-operator-sync.+|grafana-permission.+"} == 1
       for: 15m
       labels:
         area: kaas
         cancel_if_outside_working_hours: {{ include "workingHoursOnly" . }}
         severity: notify
         team: {{ include "providerTeam" . }}
         topic: managementcluster
+    - alert: AtlasJobHasNotBeenScheduledForTooLong
```

Comment:
Can we please not put team name into alert name. We are already using label for this.

Reply:
Alrighty